src/core/src/op/fake_convert.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/fake_convert.hpp"
+
+#include "itt.hpp"
+
+namespace ov {
+namespace op {
+namespace v13 {
+namespace fake_convert {
+static const std::vector<std::string>& get_valid_types() {
+    static const std::vector<std::string> valid_types{"f8e4m3", "f8e5m2"};
+    return valid_types;
+}
+}  // namespace fake_convert
+FakeConvert::FakeConvert(const ov::Output<ov::Node>& arg,
+                         const ov::Output<ov::Node>& scale,
+                         const ov::Output<ov::Node>& shift,
+                         std::string destination_type,
+                         bool apply_scale)
+    : Op({arg, scale, shift}),
+      m_destination_type(std::move(destination_type)),
+      m_apply_scale(apply_scale) {
+    constructor_validate_and_infer_types();
+}
+
+bool FakeConvert::get_apply_scale() const {
+    return m_apply_scale;
+}
+
+const std::string& FakeConvert::get_destination_type() const {
+    return m_destination_type;
+}
+
+void FakeConvert::validate_and_infer_types() {
+    OV_OP_SCOPE(v13_FakeConvert_validate_and_infer_types);
+    validate_type();
+    set_output_type(0, get_input_element_type(0), get_input_partial_shape(0));
+}
+
+std::shared_ptr<ov::Node> FakeConvert::clone_with_new_inputs(const ov::OutputVector& new_args) const {
+    OV_OP_SCOPE(v13_FakeConvert_clone_with_new_inputs);
+    OPENVINO_ASSERT(new_args.size() == 3, "Incorrect number of new arguments");
```

Comment:
As scale and shift are not mandatory, please make them optional. Otherwise I don't understand why we have `apply_scale`, but don't have `apply_shift`.

Reply:
If you make both `scale` and `shift` optional, you really need to have only `apply_scale` among attributes. Otherwise `apply_shift` is required.

Reply:
Let's make the shift an optional parameter. I would make scale as required.

Reply:
I think that we always need to apply scale, but shift can be optional.

Reply:
As discussed, in the updated version `scale` input is required and `shift` input optional.
The `apply_scale` attribute has been removed.